.github/workflows/docker_image.yml

```diff
@@ -0,0 +1,41 @@
+name: 'Get name and newest digest for docker image'
+
+env:
+  REGISTRY: ghcr.io
+
+on:
+  workflow_call:
+    inputs:
+      tool:
+        description: 'Name of the tool or "tools" for all tools'
+        required: true
+        type: string
+    outputs:
+      sc_tool:
+        description: 'Name and newest digest for the docker image of the input tool'
+        value: ${{ jobs.docker_image.outputs.sc_tool }}
```

Comment:
can you add an input for sc version as the repos can control that via a key?

Reply:
Sure added that now.
Didn't knwo the `builder.py` script gives you the latest version for each tool when using an older commit of sc as well.